### ISSUE TITLE
Wither Skeletons in The Nether don't seem to be spawning correctly.

### ISSUE THREAD

**Issue by da404lewzer** [The-1-7-10-Pack/The-1.7.10-Pack#154]
TL;DR: Wither skeletons don't seem to evenly spawn across the nether, but instead are concentrated into a single spot in my nether which causes thousands of them to accumulate.

A few days ago my console was spewing this data: http://pastebin.com/raw.php?i=xW0q12Qv (I had posted to http://www.technicpack.net/modpack/the-1710-pack.453902/changelog/18444 and was asked to post my findings here. I have since found more information on the subject.

I added opis to my version of the pack and had a good snoop. Well well, lookie here. http://i.imgur.com/2XuLKfS.png Keep in mind that we have no spawners of any kind spawning these guys. This is all vanilla spawning.

After killing all wither skeletons via an opis command a couple hundred variants of wither skeletons stuck around: http://imgur.com/a/fxeP8 I had to do this whole process twice because after leaving the nether it started happening again. To fix the remedy I put a bucked of lava in the corner of where they were piling up. Crude I know, but it's working. However..

Fast forward a couple days to today: The Nether seems to be void of all life. Ghasts and hellhound packs are the only thing that occasionally spawn excluding blazes near their spawners, but not a single freaking wither skeleton. Which sucks because we really need withers!

**Comment by The-1-7-10-Pack** [comment]
How old is your nether? Try to do a check with a new generated world.

**Comment by da404lewzer** [comment]
I started this server with version 0.6.1 and just recently upgraded to 0.7.0 the other day. I'll look into a new world tomorrow as I don't want to take this one offline so I'll have to setup a new instance. Let me know if you want connection info and I'll pm you in case you want to see for yourself or whatever you guys do lol :)

**Comment by The-1-7-10-Pack** [comment]
Do you have any additional mods?

**Comment by da404lewzer** [comment]
I haven't added anything custom to this pack or modified it in any way. Opis was only used to investigate the problem. If you haven't used it, it's a well known tool for analyzing lag and I'll be removing it after this. (Fear not, I've been running servers for years, I know my way around these bloody things.)

**Comment by da404lewzer** [comment]
Have any mods changed mob spawning behaviors that anyone is aware of?

**Comment by The-1-7-10-Pack** [comment]
Try to test what I told you there: https://github.com/The-1-7-10-Pack/The-1.7.10-Pack/issues/154#issuecomment-72421149

**Comment by da404lewzer** [comment]
I created two new servers and both of them had normal spawning. It's probably a one in a million thing knowing my luck. I guess mark it can't reproduce and I'll respond if anything changes or I find the source.

**Comment by The-1-7-10-Pack** [comment]
Closing it for now.
